app/src/main/java/org/mozilla/focus/home/HomeFragment.java

```diff
@@ -191,6 +192,7 @@ public void onCreate(Bundle bundle) {
         this.presenter = new TopSitesPresenter();
         this.presenter.setView(this);
         this.presenter.setModel(this);
+        chromeViewModel = Inject.obtainChromeViewModel(getActivity());
```

Comment:
getActivity() here might be null in fragment re-create scenario. Maybe you should consider moving this to onActivityCreated(), where getActivity() is guaranteed to be valid.